Pipfile

```diff
@@ -32,3 +32,26 @@ format = "black ."
 generate_reports = "python -m personal_finances.generate_reports"
 fetch_transactions = "python -m personal_finances.fetch_transactions"
 merge_transactions = "python -m personal_finances.merge_transactions"
+pack="""
+    bash -c ' \
+    # Generate requirements.txt from Pipenv
+    pipenv requirements > requirements.txt && \
+
+    # Create package directory and install dependencies using aws python image
+    rm -rf package && \
+    mkdir package && \
+    sudo docker run --rm -v $(pwd):/var/task --entrypoint /bin/bash public.ecr.aws/lambda/python:3.11 -c "\
```

Comment:
I'm a bit confused why we're running a pip install inside a docker container. Doesn't "pipenv" already resolve the virtual env? If there's a trick it's better to document it.

Reply:
We are running the docker because it is the most straightforward way to install the dependencies correctly. Since it uses a Docker image from Amazon, it automatically install the correct binaries for the AWS cloud hardware architecture